covalent/_workflow/transport.py

```diff
@@ -88,7 +88,7 @@ def serialize(self) -> bytes:
         )
 
     @staticmethod
-    def deserialize(data) -> "TransportableObject":
+    def deserialize(data: "TransportableObject") -> "TransportableObject":
```

Comment:
@Udayan853 I'm not a 100% convinced that the data here is of TransportableObject type. Could you try creating an instance of TransportableObject and experiment with what type of inputs the deserialize method can take? Double checking this carefully can't hurt. 

One reason why I think this might not be the data type is bcs cloudpickle is being used to load the data in ln 102. Perhaps, it's more useful to check what are the input types that cloudpickle.loads accepts.

Another point is that perhaps the description of data in args is confusing. Feel free to change it if you think this can be improved.

Reply:
Excellent!

Reply:
@FyzHsn Yeah, data should have been `bytes` type as sayandipdutta stated, it was ignorant of me to have assumed it to be `TransportableObject` blindly.
Would it be better to have the description of args as `base64 encoded cloudpickled function`?

Reply:
@Udayan853 I think just “cloud pickled function” as an arg description should be suffice. :)